proxyclient/tools/picocom-sec.sh

```diff
@@ -1,7 +1,30 @@
 #!/bin/sh
+#
+# Secondary UART device name exposed when m1n1 boots is /dev/m1n1-sec on
+# Linux host and /dev/cu.usbmodemP_03 on macOS host
+#
+DEFAULT_SECDEV=/dev/m1n1-sec
+PLATFORM_NAME=$(uname)
+
+case "${PLATFORM_NAME}" in
+Linux)
+    SECDEV=/dev/ttyACM1 ;;
```

Comment:
Same here, the udev rules make `/dev/m1n1-sec` work on Linux (more reliably than guessing ACM device numbers).

Reply:
Nice, I should have figured this out … I will remove this and add some comments about the udev stuff